lib/jmap/account_id.g.dart

```diff
@@ -6,11 +6,9 @@ part of 'account_id.dart';
 // JsonSerializableGenerator
 // **************************************************************************
 
-AccountId _$AccountIdFromJson(Map<String, dynamic> json) {
-  return AccountId(
-    const IdConverter().fromJson(json['id'] as String),
-  );
-}
+AccountId _$AccountIdFromJson(Map<String, dynamic> json) => AccountId(
```

Comment:
Do not change the `format code` in JsonSerializableGenerator `.g` files without changing its properties.